### PR TITLE
docs: add font installation guide for Lato Italic

### PR DESCRIPTION
**Pull request by chaitanyadeorukhkar**

### CODE REVIEW

**Review by divyanshu013 — APPROVED** [review]
Looks good. Doesn't auto link work on android for fonts?

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 1]
```diff
@@ -75,16 +75,18 @@ We recommend to use self hosted fonts. We'll use [Fontsource](https://fontsource
 - Import within your app entry file(eg: `App.tsx`, `entryBrowser.tsx` etc.).
   ```js
   import '@fontsource/lato/400.css';
+  import '@fontsource/open-sans/400-italic.css';
```
Comment (kamleshchandnani) [inline packages/blade/docs/guides/Installation.stories.mdx:78]: why do we need open sans?

Comment (chaitanyadeorukhkar) [reply packages/blade/docs/guides/Installation.stories.mdx]: Copy paste 🤦

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 2]
```diff
@@ -75,16 +75,18 @@ We recommend to use self hosted fonts. We'll use [Fontsource](https://fontsource
 - Import within your app entry file(eg: `App.tsx`, `entryBrowser.tsx` etc.).
   ```js
   import '@fontsource/lato/400.css';
+  import '@fontsource/lato/400-italic.css';
   import '@fontsource/lato/700.css';
+  import '@fontsource/lato/700-italic.css';
```
Comment (kamleshchandnani) [inline packages/blade/docs/guides/Installation.stories.mdx:80]: don't we need to update our typography text to accommodate this change?

Comment (chaitanyadeorukhkar) [reply packages/blade/docs/guides/Installation.stories.mdx:80]: No, it picks it up. Check the implementation of Badge.

**Review by saurabhdaware — APPROVED** [review]
Do we not have to install it somewhere in our storybook? for storybook to pickup the right font?

**Comment by chaitanyadeorukhkar** [comment]
> Do we not have to install it somewhere in our storybook? for storybook to pickup the right font?

Already done that on master

**Comment by saurabhdaware** [comment]
This can be merged right?

**Pull request merged**